faunadb/query.go

```diff
@@ -40,11 +40,14 @@ func unescapedBindings(obj Obj) unescapedObj {
 
 // Optional parameters
 
-// Events is an boolean optional parameter that describes if the query should include historical events.
+// Deprecated: That function called Events before and was renamed to support new events history api.
```

Comment:
I think this needs to be clarified. Maybe something like:

> // Deprecated: The `Events` function was renamed to `EventsOpt` to support the new history API. `EventsOpt` is provided here for backwards compatibility. Instead of using `Paginate` with the `EventsOpt` parameter, you should use the new `Events` function.